app/chat/templates/embed.html

```diff
@@ -29,6 +29,13 @@
       {% include 'home/nav.html' %}
     </div>
     <style type="text/css">
+
+      @media only screen and (max-width : 320px) {
+        .chatWrapper {
+          padding-bottom: 150% !important;
```

Comment:
not sure what is the exact intention here, but you can use 100vh (100% of the viewport height ) or even use calc(100 - size of element)

Reply:
How the wrapper works to ensure the internal contents are responsive is by extending the bottom of the wrapper div with padding, 39% gives 16:4 with a footer, but when it comes to mobile we want to extend beyond the current max height by 50%.

how would 100vh in this context?

Reply:
but then is not adding scroll to the screen?

Reply:
maybe just adding `d-none d-md-block` class to the footer so get hidden on mobile